app/actions/Inspector.js

```diff
@@ -32,11 +32,13 @@ export function bindAppium () {
         let path = (index !== undefined) && `${!parentPath ? '' : parentPath + '.'}${index}`;
 
         // Get an xpath for this element as well to use for Appium calls
-        let xpath = path && path.split('.').map((index) => `//*[${parseInt(index, 10) + 1}]`).join('');
+        let xpath = path && path.split('.').map((index) => `/*[${parseInt(index, 10) + 1}]`).join('');
```

Comment:
I'm not sure, but we may still want for the first selector to have `//` in front of it?